rpm/0049-sailfishos-gecko-Enable-the-mobile-video-control-con.patch

```diff
@@ -0,0 +1,47 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Andrew den Exter <andrew.den.exter@jolla.com>
+Date: Fri, 1 Oct 2021 06:57:53 +0000
+Subject: [PATCH] [sailfishos][gecko] Enable the mobile video control config
+ for Sailfish.
+
+---
+ toolkit/content/widgets/videocontrols.js | 9 ++++++---
+ 1 file changed, 6 insertions(+), 3 deletions(-)
+
+diff --git a/toolkit/content/widgets/videocontrols.js b/toolkit/content/widgets/videocontrols.js
+index 0b41769f680c..03e98a3c01d0 100644
+--- a/toolkit/content/widgets/videocontrols.js
++++ b/toolkit/content/widgets/videocontrols.js
+@@ -19,7 +19,8 @@ this.VideoControlsWidget = class {
+     this.document = this.element.ownerDocument;
+     this.window = this.document.defaultView;
+ 
+-    this.isMobile = this.window.navigator.appVersion.includes("Android");
++    this.isMobile = this.window.navigator.appVersion.includes("Android") ||
++                    this.window.navigator.appVersion.includes("Sailfish");
```

Comment:
This requires that start using user-agent building patch which is fine. It should ok to land this before https://github.com/sailfishos/gecko-dev/pull/61/ . Alternatively you could pick that user-agent building to this PR which would be logically good option as well. Please sync with @llewelld how to handle.

Reply:
I've changed it (back) to always be true as suggested.